client/src/components/Card/Card.css

```diff
@@ -12,57 +12,65 @@
   font-family: var(--default-font);
   min-height: 9.5rem;
 }
+
 .Card--purple {
   background-color: var(--purple);
 }
+
 .Card--pink {
   background-color: var(--pink);
 }
+
 .Card--red {
   background-color: var(--red);
 }
+
 .Card--pink {
   background-color: var(--pink);
 }
+
 .Card--yellow {
   background-color: var(--yellow);
 }
+
 .Card--large {
   width: 32vw;
   max-width: 500px;
   min-width: 200px;
   max-height: 275px;
 }
+
 .Card--medium {
   width: 14.2vw;
   max-width: 225px;
   max-height: 150px;
   min-width: 180px;
-  margin: 0 0.1vw 0 0.1vw;
+  margin: 0 0.2vw 0 0.2vw;
 }
+
 .Card--small {
   max-width: 225px;
   max-height: 125px;
 }
+
 .Card--extra-small {
   color: var(--white);
-  min-width: 180px;
+  min-width: 7rem;
+  max-width: 7rem;
+  min-height: 5rem;
+  font-size: 0.9rem;
```

Comment:
I think there are variables for the font sizes. Can you use that?

Reply:
No, I looked in the root and did not see any variables for font sizes.